CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
  - C++ interpreter API: added a `Value::Value(int)` constructor
  - Globals Singleton may now refer to other global singletons
  - Added `input-type` property to `TextInput` and `LineEdit` that allows for characters to be replaced in password fields
+ - Added `on_close_requested` function to `Window` to register callbacks that are emitted when the user tries to close a window.
```

Comment:
Since this is going to be in 0.2.2, we need to create a new unreleased section for it